lists of standard moves

zhangchicool opened the pull request:
Updated lists of standard moves.  Importantly, made the parsSPR1 and parsTBR1 for standard user.

zhangchicool committed:
white noise model

zhangchicool committed:
clean up

zhangchicool committed:
updated lists of standard moves

zhangchicool committed:
a

ronquist commented:
The diff of the proposal.c file is not computed properly making it almost impossible to review this pull request in its entirety. However, I have looked through the other files, and parts of the proposal.c file, and those changes look reasonable to me. However, it is important to describe the changes properly in the news / releases file, as some of the old moves change implementation.

zhangchicool committed:
Merge commit 'f00252fa50de4d538bdb1ceca14ad3378cdd8c8e' into feature/standard_moves

zhangchicool committed:
corrections according to Fredrik's comments

zhangchicool commented:
Basically I just changed the flags (STANDARD_USER <-> DEVELOPER) for some moves. The diff might not align properly, but I didn't change any code inside the moves. 
1. ParsSPR is still STANDARD_USER but I changed relProposalProb to 0.0
2. ParsSPR1 is now STANDARD_USER and has relProposalProb = 3.0
3. ParsTBR1 is now STANDARD_USER and has relProposalProb = 3.0

I think the details of the new moves have been described in our manuscript.

ronquist commented:
Ok to merge from me.
